Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1038,13 +1167,20 @@ public static PlayerInput FindFirstPairedToDevice(InputDevice device)
         /// <param name="prefab">Prefab to clone. Must contain a PlayerInput component somewhere in its hierarchy.</param>
         /// <param name="playerIndex">Player index to assign to the player. See <see cref="PlayerInput.playerIndex"/>.
         /// By default will be assigned automatically based on how many players are in <see cref="all"/>.</param>
-        /// <param name="controlScheme">Control scheme to activate</param>
-        /// <param name="splitScreenIndex"></param>
+        /// <param name="controlScheme">Control scheme to activate.</param>
+        /// <param name="splitScreenIndex">Which split screen to instantiate on.</param>
         /// <param name="pairWithDevice">Device to pair to the user. By default, this is <c>null</c> which means
         /// that PlayerInput will automatically pair with available, unpaired devices based on the control schemes (if any)
         /// present in <see cref="actions"/> or on the bindings therein (if no control schemes are present).</param>
-        /// <returns></returns>
+        /// <returns>Newly created PlayerInput component</returns>
```

Comment:
Missing full stop

Reply:
<c> formatting missing it seems

Reply:
Might add ". Never <c>null</c>." since the user then knows they do not need to null check. In lack of nullable syntax.

Reply:
Fixing full stop and formatting in next commit
It looks like it could be null so I won't add that part.